Code/RDBoost/Wrap.h

```diff
@@ -18,6 +18,10 @@
 //
 #include <boost/python.hpp>
 #include <boost/python/stl_iterator.hpp>
+#include <memory>
+// ha ha... we don't actually want to use this, but we need to keep support old
+// C++ versions:
+#define rdk_unique_ptr std::auto_ptr
```

Comment:
I don't think auto_ptr should be used.  I think something like:

````
if CXX11
  typedef std::unique_ptr rdk_unique_ptr ;
elif BOOSTVERSION < 1.59
  typedef boost::shared_ptr rdk_unique_ptr;
else
  typedef boost::unique_ptr rdk_unique_ptr;
#endif
```

This unfortunately means that pre boost 1.59 will have some performance penalties, but I believe this is more forward thinking.